examples/dt/perfscale/scalelab/nncp/values.yaml

```diff
@@ -0,0 +1,221 @@
+        cidr: 172.20.0.0/16
+        name: subnet1
+        vlan: 20
+    mtu: 1500
+    prefix-length: 24
+    iface: storage
+    vlan: 20
+    base_iface: ens2f1np1
+    lb_addresses:
+      - 172.20.2.10-172.20.2.250
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "storage",
+        "type": "macvlan",
+        "master": "storage",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.20.0.0/16",
+          "range_start": "172.20.1.10",
+          "range_end": "172.20.1.250"
+        }
+      }
+  tenant:
+    dnsDomain: tenant.example.com
+    subnets:
+      - allocationRanges:
+          - end: 172.18.254.254
+            start: 172.18.5.100
+        cidr: 172.18.0.0/16
+        name: subnet1
+        vlan: 18
+    mtu: 1500
+    prefix-length: 24
+    iface: tenant
+    vlan: 18
+    base_iface: ens2f0np0
+    lb_addresses:
+      - 172.18.2.10-172.18.2.250
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "tenant",
+        "type": "macvlan",
+        "master": "tenant",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.18.0.0/16",
+          "range_start": "172.18.1.10",
+          "range_end": "172.18.1.250"
+        }
+      }
+  datacentre:
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "datacentre",
+        "type": "bridge",
+        "bridge": "ospbr",
+        "ipam": {}
+      }
+  ctlplane:
+    dnsDomain: ctlplane.example.com
+    subnets:
+      - allocationRanges:
+          - end: 172.16.254.254
+            start: 172.16.20.100
+        cidr: 172.16.0.0/16
+        gateway: 172.16.0.1
+        name: subnet1
+    prefix-length: 24
+    iface: ens1f1
+    mtu: 9000
+    lb_addresses:
+      - 172.16.16.10-172.16.16.250
+    endpoint_annotations:
+      metallb.universe.tf/address-pool: ctlplane
+      metallb.universe.tf/allow-shared-ip: ctlplane
+      metallb.universe.tf/loadBalancerIPs: 172.16.16.120
+    net-attach-def: |
+      {
+        "cniVersion": "0.3.1",
+        "name": "ctlplane",
+        "type": "macvlan",
+        "master": "ens1f1",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.16.0.0/16",
+          "range_start": "172.16.15.10",
+          "range_end": "172.16.15.250"
+        }
+      }
+  dns-resolver:
+    config:
+      server:
+        - 192.168.122.1
+      search: []
+    options:
+      - key: server
+        values:
+          - 10.1.37.190
+      - key: server
+        values:
+          - 172.16.0.1
+
+  routes:
+    config: []
+
+  rabbitmq:
+    endpoint_annotations:
+      metallb.universe.tf/address-pool: internalapi
+      metallb.universe.tf/loadBalancerIPs: 172.17.2.125
+  rabbitmq-cell1:
+    endpoint_annotations:
+      metallb.universe.tf/address-pool: internalapi
+      metallb.universe.tf/loadBalancerIPs: 172.17.2.126
+
+  lbServiceType: LoadBalancer
+  storageClass: lvms-vg1
+  bridgeName: ens1f1
```

Comment:
Using a single NIC interface as `bridgeName` is probably not the right fit in the long run.  We need to refactor the entire architecture repo to remove control plane bridges [1], at which point this field will go away.

[1] https://issues.redhat.com/browse/OSPRH-18204